Library/Homebrew/cmd/info.rb

```diff
@@ -68,6 +69,20 @@ def info
   end
 
   def print_info
+    if args.days.present?
+      valid_days = %w[30 90 365]
```

Comment:
Might be worth making all these arrays class-wide constants and `.freeze`ing them.